README.md

```diff
@@ -49,8 +49,8 @@ Links in [contributors/devel/README.md](contributors/devel/README.md)
 lead to many relevant topics, including
  * [Developer's Guide] - how to start a build/test cycle
  * [Collaboration Guide] - how to work together
- * [expectations] - what the community expects
- * [pull request] policy - how to prepare a pull request
+ * [Expectations] - what the community expects
+ * [Pull Request] policy - how to prepare a pull request
```

Comment:
Can you drop the word "policy" and change the link text to Pull Request Process? Lines up with the linked doc title that way